src/results.html

```diff
@@ -0,0 +1,78 @@
+            <div class="card"> <!-- Do fields need to be divs instead? -->
+                <h2 class="name">Dr: </h2>
+                <h3 class="credentials">Credentials</h3>
+                <h3 class="gender">Gender:</h3>
+                <h3 class="location">Location:</h3>
+                <ul><h3 id="taxTitle">Taxonomies:</h3>
+                    <li class="taxonomies 1">family medicine</li>
+                    <li class="taxonomies 2">chiropractor</li>
+                    <li class="taxonomies 3">chiropractor</li>
+                </ul>
+                <h4 class="lastUpdated">Last Updated: 12/21/2020</h4>
+                <button type="button" class="reviews">See Reviews</button>
+            </div>
+            <!-- Use CreateCard() script to create cards -->
+        </div>
+
+        <div id="copy">
+            <h2 class="name">Dr: </h2>
+            <h3 class="credentials">Credentials:</h3>
+            <h3 class="gender">Gender:</h3>
+            <h3 class="location">Location:</h3>
+            <ul><h3 class="TaxonomiesTitle">Taxonomies</h3>
+                <li class="taxonomies 1">family medicine</li>
+                <li class="taxonomies 2">chiropractor</li>
+                <li class="taxonomies 3">chiropractor</li>
+            </ul>
+            <h4 class="lastUpdated">Last Updated: 12/21/2020</h4>
+            <button type="button" class="reviews">See Reviews</button>
+        </div>
+
+        <script src="index.js">
+            createCards(6); // Create a specific number of cards
+
+            var cards = document.getElementsByClassName("card");
+            for(let i of cards){
+                var cardName = i.getElementsByTagName("h2");
+                cardName.innerHTML = "Dr. York"
+            } // not working yet
+
+        //     var cards = document.getElementsByClassName("card");
+        //     var name = document.getElementsByClassName("name"); 
+        //     var creds = document.getElementsByClassName("credentials");
+        //     var gender = document.getElementsByClassName("gender");
+        //     var location = document.getElementsByClassName("location");
+        //     var taxonomies = document.getElementsByClassName("taxonomies"); // Are we getting 1,2,3
+        //     var lastupdated = document.getElementsByClassName("lastUpdated");
+        //     var i;
+        //     function addDoctor(){ // Need doctors information here
+        //         for (i = 0; i < cards.length; i++){ 
+        //             name[i].innerHTML = "Dr. " +"";
+        //             creds[i].innerHTML = "Credentials: " +"";
+        //             gender[i].innerHTML= "Gender:" +"";
+        //             location[i].innerHTML = "Location:" +"";
+        //             taxonomies[0].innerHTML; taxonomies[1].innerHTML;taxonomies[2].innerHTML;
+        //             lastupdated[i].innerHTML = "Last Updated: " + "";           
+        //         }
+        //     }
+        </script>
```

Comment:
1. The reason this is not working is that you are using the same script tag to import and to define local scripts. Instead, use two tags: <script src="index.js"></script><script>Custom JS</script>.
2. Instead of copying a card and then querying the DOM, it may be easier to use a formatting string and loop to dynamically create the cards like so: 
```
let info = [ { name: 'A', credential: 1 },
             { name: 'B', credential: 2} 
           ];
const reducer = (aggregatingValue, {name, credential}) => aggregatingValue + 
        `<div class="card">
                    <h2 class="name">Dr: ${name}</h2>
                    <h3 class="credentials">Credentials: ${credential}</h3>
                    <h3 class="gender">Gender:</h3>
                    <h3 class="location">Location:</h3>
                    <ul><h3 class="TaxonomiesTitle">Taxonomies</h3>
                            <li class="taxonomies 1">family medicine</li>
                            <li class="taxonomies 2">chiropractor</li>
                            <li class="taxonomies 3">chiropractor</li>
                   </ul>
                   <h4 class="lastUpdated">Last Updated: 12/21/2020</h4>
                   <button type="button" class="reviews">See Reviews</button> 
        </div>`;
document.getElementById("results_container").innerHTML = info.reduce(reducer, '');
```